lib/monitoring/aws-rds/RdsClusterMonitoring.ts

```diff
@@ -43,6 +45,13 @@ export interface RdsClusterMonitoringOptions extends BaseMonitoringProps {
     string,
     HighConnectionCountThreshold
   >;
+  readonly addReadIOPSAlarm?: Record<string, MaxUsageCountThreshold>;
+  readonly addWriteIOPSAlarm?: Record<string, MaxUsageCountThreshold>;
+  readonly addSelectLatencyAlarm?: Record<string, LatencyThreshold>;
+  readonly addInsertLatencyAlarm?: Record<string, LatencyThreshold>;
+  readonly addUpdateLatencyAlarm?: Record<string, LatencyThreshold>;
+  readonly addDeleteLatencyAlarm?: Record<string, LatencyThreshold>;
+  readonly addCommitLatencyAlarm?: Record<string, LatencyThreshold>;
```

Comment:
Should update https://github.com/cdklabs/cdk-monitoring-constructs/blob/main/test/monitoring/aws-rds/RdsClusterMonitoring.test.ts to reflect these too.